stingray/lightcurve.py

```diff
@@ -94,6 +103,10 @@ def __init__(self, time, counts, input_counts=True):
 
         self.tseg = self.time[-1] - self.time[0] + self.dt
         self.tstart = self.time[0] - 0.5*self.dt
+        self.gti = \
+            np.asarray(assign_value_if_none(gti,
+                                            [[self.tstart,
+                                              self.tstart + self.tseg]]))
```

Comment:
Done. Instead of `apply_gti`, I called it `split_lc_by_gti`, that in my opinion better describes what is actually done.